src/main/java/pageObjects/Cart.java

```diff
@@ -6,38 +6,38 @@
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 
-import AbstractComponents.AbstractComponents;
+import commonMethods.CommonMethods;
  
-public class Cart extends AbstractComponents{						// Cart in the Amazon website
+public class Cart extends CommonMethods{						// Cart in the Amazon website
 WebDriver driver; 
  
 public Cart(WebDriver driver)
 {
 	super(driver);
 	this.driver=driver; 
 }
-By addtocheckoutbutton = By.name("proceedToRetailCheckout");
+By addToCheckoutButton = By.name("proceedToRetailCheckout");
 By confirmation = By.xpath("//*[text()=\" was removed from Shopping Cart. \"]");
-public Checkoutpage addToCheckout()															// clicking the checkout button
+public CheckoutPage addToCheckout()															// clicking the checkout button
 {  
-	driver.findElement(addtocheckoutbutton).click();
-	return new Checkoutpage(driver);														//returning the next class constructor as return statement 
+	driver.findElement(addToCheckoutButton).click();
+	return new CheckoutPage(driver);														//returning the next class constructor as return statement 
 }
 public List<WebElement> getCartList()
 {
  List<WebElement> cartlist = driver.findElements(By.xpath("//div[@data-bundleitem=\"absent\"]"));
 return cartlist;
 } 
-public void selectItem(String productname) throws InterruptedException  			//finding the exact product to delete from the string using Streams() function 
+public void selectItem(String product) throws InterruptedException  			//finding the exact product to delete from the string using Streams() function 
 {		
-	WebElement item = getCartList().stream().filter(s->s.findElement(By.cssSelector("span[class='a-truncate-cut']")).getText().contains(productname)).findFirst().orElse(null);		//selecting the desired product
+	WebElement item = getCartList().stream().filter(s->s.findElement(By.cssSelector("span[class='a-truncate-cut']")).getText().contains(product)).findFirst().orElse(null);		//selecting the desired product
 	Thread.sleep(2000);
 	WebElement selectedElement = item.findElement(By.xpath("div[@class='sc-list-item-content']//div//div[@class='sc-item-content-group']/child::div//span[@data-feature-id=\"delete\"]//span//input"));	//clicking the delete button within the selected item
```

Comment:
xpath isn't bad but would look for one that is a little cleaner or a little more concise if possible

Reply:
yeah, that xpath  was the only solution from my end,  I mean, how to select that delete button in each item, Sure will  try for a better one